core/templates/services/context.service.spec.ts

```diff
@@ -160,6 +204,17 @@ describe('Context service', () => {
       () => {
         expect(ecs.canAddOrEditComponents()).toBe(true);
       });
+
+    it('should correctly affirm that page contains reference to skills',
+      () => {
+        expect(ecs.getPageContext()).toBe('topic_editor');
+        expect(ecs.canEntityReferToSkills()).toBe(true);
+      });
+
+    it('should affirm the page context to be exploration player or editor',
+      ()=> {
+        expect(ecs.isInExplorationContext()).toBe(false);
```

Comment:
What action do we do here that sets the exploration context to be false?

Reply:
`spyOn(urlService, 'getPathname').and.returnValue('/topic_editor/123');` in beforeEach is cause of this. When getPageContext in above spec block sets it to topic editor. Now isInExplorationContext is true when pageContext is exploration player or editor. Else, its false.